backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -457,6 +471,17 @@ public void deleteOrganization(Organization org) {
     return getOrganizationRoleClaimsForUser(user);
   }
 
+  private Optional<OrganizationRoleClaims> getOrganizationRoleClaimsFromTenantDataAccess(
+      Collection<String> groupNames) {
+    List<OrganizationRoleClaims> claims = _extractor.convertClaims(groupNames);
+
+    if (claims.size() != 1) {
+      LOG.warn("User is in {} Okta organizations, not 1", claims.size());
```

Comment:
**question (non-blocking):** What would cause this edge case?

Reply:
This is here to be consistent with the checks we perform before returning role claims from other methods.  As it is right now, it wouldn't happen unless the values in the database are modified.

Reply:
Got it, thanks!